<?xml version="1.0" encoding="utf-8"?>
<doc xmlns:dev="http://ddue.schemas.microsoft.com/authoring/2003/5" xmlns:xlink="http://www.w3.org/1999/xlink" fileAssetGuid="c8cc7952-b140-4231-8260-f63bb6f3a251" revisionNumber="4">
  <assembly>
    <name>mscorlib</name>
  </assembly>
  <members>
    <member name="T:System.NotCancelableException">
      <dduexml xmlns="http://ddue.schemas.microsoft.com/authoring/2003/5">
        <summary>
          <para>The exception that is thrown when a thread attempts to cancel a synchronous operation that cannot be cancelled.</para>
        </summary>
        <remarks>
          <content>
            <para>This exception is thrown when a synchronous operation is cancelled on an object that is not associated with a cancellation region, is associated with a cancellation region that is noncancelable, or is associated with a cancellation region that has been disposed.</para>
          </content>
        </remarks>
      </dduexml>
    </member>
    <member name="Overload:System.NotCancelableException.#ctor">
      <dduexml xmlns="http://ddue.schemas.microsoft.com/authoring/2003/5">
        <summary>
          <para>Initializes a new instance of the <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference> class.</para>
        </summary>
        <remarks>
          <content>
          </content>
        </remarks>
      </dduexml>
    </member>
    <member name="M:System.NotCancelableException.#ctor">
      <dduexml xmlns="http://ddue.schemas.microsoft.com/authoring/2003/5">
        <summary>
          <para>Initializes a new instance of the <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference> class with a system-supplied error message.</para>
        </summary>
        <remarks>
          <content>
            <para>This constructor initializes the <codeEntityReference autoUpgrade="true">P:System.Exception.Message</codeEntityReference> property of the new instance to a system-supplied message that describes the error, such as "The operation attempted to be canceled could not be because it was not in a cancelable region." This message takes into account the current system culture.</para>
            <para>The following table shows the initial property values for an instance of <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference>.</para>
            <table>
              <tableHeader>
                <row>
                  <entry>
                    <para>Property </para>
                  </entry>
                  <entry>
                    <para>Value </para>
                  </entry>
                </row>
              </tableHeader>
              <row>
                <entry>
                  <para>
                    <codeEntityReference autoUpgrade="true">P:System.Exception.InnerException</codeEntityReference>
                  </para>
                </entry>
                <entry>
                  <para>A null reference (<languageKeyword>Nothing</languageKeyword> in Visual Basic). </para>
                </entry>
              </row>
              <row>
                <entry>
                  <para>
                    <codeEntityReference autoUpgrade="true">P:System.Exception.Message</codeEntityReference>
                  </para>
                </entry>
                <entry>
                  <para>The localized error message string. </para>
                </entry>
              </row>
            </table>
          </content>
        </remarks>
        <relatedTopics>
          <codeEntityReference autoUpgrade="true">T:System.Exception</codeEntityReference>
          <link xlink:href="F99A1D29-A2A8-47AF-9707-9909F9010735">Handling and Throwing Exceptions</link>
        </relatedTopics>
      <overload>
        <summary>
          <para>Initializes a new instance of the <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference> class.</para>
        </summary>
        <remarks>
          <content>
          </content>
        </remarks>
      </overload>
</dduexml>
    </member>
    <member name="M:System.NotCancelableException.#ctor(System.String)">
      <dduexml xmlns="http://ddue.schemas.microsoft.com/authoring/2003/5">
        <summary>
          <para>Initializes a new instance of the <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference> class with a specified error message.</para>
        </summary>
        <parameters>
          <parameter>
            <parameterReference>message</parameterReference>
            <content>
              <para>A <codeEntityReference autoUpgrade="true">T:System.String</codeEntityReference> that describes the error.</para>
            </content>
          </parameter>
        </parameters>
        <remarks>
          <content>
            <para>The content of <parameterReference>message</parameterReference> is intended to be understood by humans. The caller of this constructor is required to ensure that this string has been localized for the current system culture.</para>
            <para>The following table shows the initial property values for an instance of <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference>.</para>
            <table>
              <tableHeader>
                <row>
                  <entry>
                    <para>Property </para>
                  </entry>
                  <entry>
                    <para>Value </para>
                  </entry>
                </row>
              </tableHeader>
              <row>
                <entry>
                  <para>
                    <codeEntityReference autoUpgrade="true">P:System.Exception.InnerException</codeEntityReference>
                  </para>
                </entry>
                <entry>
                  <para>A null reference (<languageKeyword>Nothing</languageKeyword> in Visual Basic). </para>
                </entry>
              </row>
              <row>
                <entry>
                  <para>
                    <codeEntityReference autoUpgrade="true">P:System.Exception.Message</codeEntityReference>
                  </para>
                </entry>
                <entry>
                  <para>The error message string. </para>
                </entry>
              </row>
            </table>
          </content>
        </remarks>
        <relatedTopics>
          <codeEntityReference autoUpgrade="true">T:System.Exception</codeEntityReference>
          <link xlink:href="F99A1D29-A2A8-47AF-9707-9909F9010735">Handling and Throwing Exceptions</link>
        </relatedTopics>
      </dduexml>
    </member>
    <member name="M:System.NotCancelableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
      <dduexml xmlns="http://ddue.schemas.microsoft.com/authoring/2003/5">
        <summary>
          <para>Initializes a new instance of the <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference> class with serialized data.</para>
        </summary>
        <parameters>
          <parameter>
            <parameterReference>info</parameterReference>
            <content>
              <para>The object that holds the serialized object data. </para>
            </content>
          </parameter>
          <parameter>
            <parameterReference>context</parameterReference>
            <content>
              <para>The contextual information about the source or destination. </para>
            </content>
          </parameter>
        </parameters>
        <remarks>
          <content>
            <para>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see <link xlink:href="832AC524-21BC-419A-A27B-CA8BFC45840F">XML and SOAP Serialization</link></para>
          </content>
        </remarks>
        <relatedTopics>
          <link xlink:href="832AC524-21BC-419A-A27B-CA8BFC45840F">XML and SOAP Serialization</link>
        </relatedTopics>
      </dduexml>
    </member>
    <member name="M:System.NotCancelableException.#ctor(System.String,System.Exception)">
      <dduexml xmlns="http://ddue.schemas.microsoft.com/authoring/2003/5">
        <summary>
          <para>Initializes a new instance of the <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference> class with a specified error message and a reference to the inner exception that is the cause of this exception.</para>
        </summary>
        <parameters>
          <parameter>
            <parameterReference>message</parameterReference>
            <content>
              <para>The error message that explains the reason for the exception. </para>
            </content>
          </parameter>
          <parameter>
            <parameterReference>innerException</parameterReference>
            <content>
              <para>The exception that is the cause of the current exception. If the <parameterReference>innerException</parameterReference> parameter is not a null reference (<languageKeyword>Nothing</languageKeyword> in Visual Basic), the current exception is raised in a <languageKeyword>catch</languageKeyword> block that handles the inner exception. </para>
            </content>
          </parameter>
        </parameters>
        <remarks>
          <content>
            <para>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <codeEntityReference autoUpgrade="true">P:System.Exception.InnerException</codeEntityReference> property. The <codeEntityReference autoUpgrade="true">P:System.Exception.InnerException</codeEntityReference> property returns the same value that is passed into the constructor, or <languageKeyword>null</languageKeyword> if the <codeEntityReference autoUpgrade="true">P:System.Exception.InnerException</codeEntityReference> property does not supply the inner exception value to the constructor.</para>
            <para>The following table shows the initial property values for an instance of <codeEntityReference autoUpgrade="true">T:System.NotCancelableException</codeEntityReference>.</para>
            <table>
              <tableHeader>
                <row>
                  <entry>
                    <para>Property Type </para>
                  </entry>
                  <entry>
                    <para>Condition </para>
                  </entry>
                </row>
              </tableHeader>
              <row>
                <entry>
                  <para>
                    <codeEntityReference autoUpgrade="true">P:System.Exception.InnerException</codeEntityReference>
                  </para>
                </entry>
                <entry>
                  <para>The inner exception reference. </para>
                </entry>
              </row>
              <row>
                <entry>
                  <para>
                    <codeEntityReference autoUpgrade="true">P:System.Exception.Message</codeEntityReference>
                  </para>
                </entry>
                <entry>
                  <para>The error message string. </para>
                </entry>
              </row>
            </table>
          </content>
        </remarks>
        <relatedTopics>
          <codeEntityReference autoUpgrade="true">T:System.Exception</codeEntityReference>
          <link xlink:href="F99A1D29-A2A8-47AF-9707-9909F9010735">Handling and Throwing Exceptions</link>
        </relatedTopics>
      </dduexml>
    </member>
  </members>
</doc>